### DISCUSSION TITLE
I am getting "randomness.move" error while compiling the code.

### DISCUSSION THREAD

**Discussion by Nishu0** [aptos-labs/aptos-developer-discussions#79] [Questions] [answered]
Whenever I am trying to compile my code. I am getting this error. 
![image](https://github.com/aptos-labs/aptos-developer-discussions/assets/89217455/05727551-63ea-4840-a58f-67f7a53eeaed)

**Comment by Alexzander-Stone** [comment, marked as answer]
Hi @Nishu0, it seems another user had a similar issue found in Discussion post #65.

If the resolution on #65 doesn't solve your problem, please let us know!
